src/Foundation/NSUndoManager.cs

```diff
@@ -17,5 +19,23 @@ public partial class NSUndoManager {
 		public virtual void SetActionName (string actionName) {
 			SetActionname (actionName);
 		}
+
+#if NET
+		public NSRunLoopMode []? RunLoopModes {
+			get {
+				var modes = WeakRunLoopModes;
+				if (modes is null)
+					return null;
```

Comment:
You can use `return Array.Empty<NSRunLoop> ();` and remove the `?` in the signature (and the need for the caller to deal with a possibly `null` return value).